Sources/OpenAI/Public/Models/Chat/Message.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  Message.swift
+//  
+//
+//  Created by Federico Vitale on 14/11/23.
+//
+
+import Foundation
+
+public struct Message: Codable, Equatable {
```

Comment:
This is a pretty generic name for a type - I found it to be a problem in my project that already had its own `Message` type defined, and `OpenAI.Message` didn't work because the `OpenAI` module also contains a class called `OpenAI` ([see this discussion on this potential flaw in Swift](https://forums.swift.org/t/fixing-modules-that-contain-a-type-with-the-same-name/3025/25))

The only thing that worked for me was renaming the type in my own project so `Message` was available for this package to use.

Reply:
Good point, I think it would be a good idea to use some naming pattern, to avoid confusions, though this would mean that all projects using this package would need to update namings.